app/views/users/index.html.erb

```diff
@@ -1,6 +1,19 @@
-<h1>List of Users with Post Count:</h1>
-<ul>
-  <% @users.each do |user| %>
-    <li><%= link_to user.name, user_path(user) %> - <%= user.posts_counter %> posts</li>
-  <% end %>
-</ul>
+<section class="users-container">
+  <h2>List of Users with Post Count:</h2>
+
+  <div class="users-list">
+    <% @users.each do |user| %>
+      <%= link_to user_path(user) do %>
+        <div class="user">
+          <div class="user-photo">
+            <img src="<%= user.photo %>">
+          </div>
+          <ul class="user-info">
+            <li class="user-name"><%= user.name %></li>
+            <li class="post-count"><%= user.posts_counter %> posts</li>
+          </ul>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
```

Comment:
_Image is displayed with the correct URL ✨_
![image](https://github.com/christianonoh/beyond-borders/assets/106119105/9ef3dbd5-8558-4a97-9976-114297e0acdb)


- [OPTIONAL] Good job for displaying the user's photo ✨📌 Sometimes, the `URL` provided for the image is not correct and as a result a broken image is displayed 😓 To avoid this behavior and check if the link the user entered for an image is valid, you can use Ruby's `Net::HTTP` library to perform an HTTP request to the provided URL and check if the response indicates a successful request. If the response is successful, then the image is displayed; otherwise, the default image is shown. 🙌📌

Reply:
@christianbiring1 Thank you for your valuable feedback! I truly appreciate your input, and I've already implemented the changes you suggested. If you happen to have a moment, I would be extremely grateful if you could take a look and let me know if the modifications meet your expectations. Your expertise is highly valued, and I want to ensure everything is up to the mark. Thank you once again!

Reply:
Kudos @christianonoh :tada: